src/libraries/System.Private.CoreLib/src/System/Globalization/TimeSpanParse.cs

```diff
@@ -1324,18 +1321,18 @@ private static bool TryParseByFormat(ReadOnlySpan<char> input, ReadOnlySpan<char
 
                     case '\'':
                     case '\"':
-                        StringBuilder enquotedString = StringBuilderCache.Acquire();
-                        if (!DateTimeParse.TryParseQuoteString(format, i, enquotedString, out tokenLen))
+                        var enquotedString = new ValueStringBuilder(32);
```

Comment:
would it make sense to be slightly less conservative? 64?

Reply:
sure